services/leafSpaceIntegration/contactreservation/main.go

```diff
@@ -0,0 +1,12 @@
+package contactreservation
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/provider"
+)
+
+var LeafSpaceContactReservation = provider.DataSourceType[contactReservation, *contactReservation]{
+	ResourceIdentifier: "leanspace_leaf_space_contact_reservation",
+	Path:               "integration-leafspace/contact-reservations/status/mappings",
+	Schema:             contactReservationSchema,
+	FilterSchema:       nil,
```

Comment:
I think you can provide the database filter schema to have the leafspaceStatuses, right?